conans/test/integration/build_environment_test.py

```diff
@@ -149,6 +149,46 @@ def build(self):
         expected_curdir_base = unix_path(client.client_cache.conan(ConanFileReference.loads("bash/0.1@lasote/stable")))
         self.assertIn(expected_curdir_base, client.user_io.out)
 
+    def run_in_windows_bash_relative_path_test(self):
+        if platform.system() != "Windows":
+            return
+        conanfile = '''
+from conans import ConanFile, tools
+
+class ConanBash(ConanFile):
+    name = "bash"
+    version = "0.1"
+
+    def build(self):
+        tools.mkdir("relative")
+        tools.run_in_windows_bash(self, "pwd", "relative")
+'''
+        client = TestClient()
+        client.save({CONANFILE: conanfile})
+        client.run("create bash/0.1@lasote/stable")
+        self.assertIn("build/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9/relative", client.user_io.out)
+
+    def run_in_windows_bash_env_var_test(self):
+        if platform.system() != "Windows":
+            return
+        conanfile = '''
+from conans import ConanFile, tools
+
+class ConanBash(ConanFile):
+    name = "bash"
+    version = "0.1"
+    settings = "os", "compiler", "build_type", "arch"
+
```

Comment:
The issue is when there is a "requires" clause, for example cmake: its path is present in ``` self.run("echo %PATH%")``` but not in ```tools.run_in_windows_bash(self, "echo $PATH")```

Reply:
Thanks, I reproduced the issue now.